test/controllers/application_controller_test.rb

```diff
@@ -108,7 +108,7 @@
 
     it 'reports to newrelic on FisbotApiError' do
```

Comment:
Change newrelic comment to datadog

Reply:
Good find. I grepped for newrelic and found even more places to edit.